pkg/plugins/runtime/k8s/metadata/annotations.go

```diff
@@ -11,11 +11,6 @@ import (
 
 // Annotations that can be used by the end users.
 const (
-	// KumaMeshAnnotation defines a Pod annotation that
-	// associates a given Pod with a particular Mesh.
-	// Annotation value must be the name of a Mesh resource.
```

Comment:
i kind of think it'd be nice to keep this, mark it deprecated and then use it at the one place where we check Annotations

Reply:
and also in the tests. this way it's clear that we have something deprecated still